src/applications/dhp-connected-devices/components/DevicesToConnectSection.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { DeviceConnectionCard } from './DeviceConnectionCard';
+
+export const DevicesToConnectSection = ({ connectedDevices }) => {
+  const areAllDevicesConnected = () => {
+    return connectedDevices.every(device => device.connected);
+  };
+  const devicesToConnectMapped = () => {
+    return connectedDevices.map(device => {
+      if (!device.connected) {
+        return <DeviceConnectionCard key={device.vendor} device={device} />;
+      }
+      return <></>;
+    });
+  };
+
+  return (
+    <>
+      {areAllDevicesConnected() && (
+        <p data-testId="all-devices-connected-alert">
+          You have connected all supported devices
```

Comment:
I may have an outdated reference for wire frames, but what I see on mine is different than here.

I see some kind of H tag; "Devices you can connect" and underneath it:
"There are no devices able to connect"

Reply:
Let's use, "There are no devices available to connect."